GEOPY-2745: Refresh screenshots of UIJsons in docs (#66)
MiraGeoscience/grid-apps: docs/source/block_model_to_octree.rst

Block Model to Octree
=====================

.. figure:: /images/block_model_to_octree_uijson.png
   :width: 500

> The ``Block model to Octree`` module has been developed as a user-interface for the conversion between regular (Tensor) BlockModel objects to Octree, using the `discretize <https://discretize.simpeg.xyz/en/main/>`_ package.

Reviewer: ```suggestion
The ``Block model to Octree`` module has been developed as a user-interface for the conversion between regular (Tensor) BlockModel objects to Octree, using the `discretize <https://discretize.simpeg.xyz/en/main/>`_ package.

.. figure:: /images/block_model_to_octree_uijson.png
   :width: 500
   
```

<img width="800" height="479" alt="Image" src="https://github.com/user-attachments/assets/4c71cc9e-6749-4acb-ab36-1229162f839d" />

Author: done

> In its most simple form, the application will create an octree with the core region matching the cells of the block model with a small padding region where the cells are allowed to expand.

Reviewer: ```suggestion
In its simplest form, the application will create an octree with the core region matching the cells of the block model. Cells are allowed to expand to the closest octree level within the volume of padding cells.
```

Author: done

.. figure:: /images/block_model_to_octree_basic_usage.png
    :width: 800

There is also an option to refine the octree on a specified model.  This will create an octree that is refined in areas that contain large gradient in the provided model.

.. figure:: /images/block_model_to_octree_advanced.png

>     :width: 800

Reviewer: Sorry to be picky, but can we use a ReferencedData instead? The refinement on the boundaries will be much clearer.

Author: done

Block Model to Octree
=====================

> The ``Block model to Octree`` module has been developed as a user-interface for the conversion between

Reviewer: ```suggestion
The ``Block Model to Octree`` module has been developed as a user interface for the conversion between
```

> regular (Tensor) BlockModel objects to Octree, using the `discretize <https://discretize.simpeg.xyz/en/main/>`_

Reviewer: ```suggestion
regular 3D grid (`BlockModel`) objects to Octree, using the `discretize <https://discretize.simpeg.xyz/en/main/>`_
```
